operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -485,6 +485,31 @@ rbac:
     runAsUser: 10001
     seccompProfile:
       type: RuntimeDefault
+  
+  # -- PSP configuration
+  podSecurityPolicy:
```

Comment:
do you think we can put this under `rbac`? This is the place which enables the PSP and configured the pod security context. I think it will be easier if this is closer to there